fdbserver/fdbserver.actor.cpp

```diff
@@ -1426,6 +1427,11 @@ int main(int argc, char* argv[]) {
 		}
 		if (!serverKnobs->setKnob("server_mem_limit", std::to_string(memLimit))) ASSERT(false);
 
+		if (EvictablePageCache::RANDOM != EvictablePageCache::evictionPolicyStringToEnum(flowKnobs->CACHE_EVICTION_POLICY) &&
```

Comment:
Since evictionPolicyStringToEnum() will throw a specific error if the string passed is not a known eviction policy it is sufficient to just call
`EvictablePageCache::evictionPolicyStringToEnum(flowKnobs->CACHE_EVICTION_POLICY)` here.

Reply:
@satherton: right. Updated the pull request with the reduced check, as you suggested.